base/io.jl

```diff
@@ -842,14 +842,13 @@ read(s::IO, T::Type) = error("The IO stream does not support reading objects of
 
 ## high-level iterator interfaces ##
 
-struct EachLine
-    stream::IO
+struct EachLine{IOT <: IO}
+    stream::IOT
     ondone::Function
     keep::Bool
-
-    EachLine(stream::IO=stdin; ondone::Function=()->nothing, keep::Bool=false) =
-        new(stream, ondone, keep)
 end
+EachLine(stream::IO=stdin; ondone::Function=()->nothing, keep::Bool=false) =
+        EachLine(stream, ondone, keep)
```

Comment:
This line should be unindented one level as well.

Reply:
I'll try fix it before merging with ci skip to not waste CI.

Reply:
I think this should remain inside the `struct` block so we don't expose a 3-argument `EachLine` method.

Reply:
Fixed